README.md

```diff
@@ -1 +1,174 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [setup and mobile first project] 
+
+This is my first project on Microverse 
+I add a heading to my html file 
+and styled it , by giving it a color.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features 
+
+> Describe between 1-3 key features of the application.
+
+- **[Add HTML]**
+- **[Add css]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://dorcas126.github.io/Setup-and-mobile-first-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started 
+
+To get a local copy up and running, follow these steps.
+just visit my repo folder (https://github.com/Dorcas126/Setup-and-mobile-first-project/tree/mobile-first)
+
+### Prerequisites
+
+In order to run this project you need:
+
+any code editor 
+but mostly use Vs code
+
+### Setup 
+
+Clone this repository to your desired folder:
+https://github.com/Dorcas126/Setup-and-mobile-first-project.git
+
+### Install
+
+Install this project with:
+
+Github
+Vs code 
+to be able to access the code
+
+### Usage
+
+To run the project, execute the following command:
+
+Linters
+git commands
+
+### Run tests
+
+To run tests, run the following command:
+
+Linters
+
+<!-- AUTHORS -->
+
+## 👥 Authors 
+
+👤 **Author1**
+
+- GitHub: [@Dorcas126](https://github.com/Dorcas126)
+- Twitter: [@Dorcasuw](https://twitter.com/Dorcasuwi)
+- LinkedIn: [Uwimbabazi Dorcas](www.linkedin.com/in/
+uwimbabazi-dorcas-956a5a226)
+
```

Comment:
Please kindly fix the issue with the link to your `LinkedIn` profile. it is not working correctly. 🛠💪

- It looks like there is a break in between the `url` whereas everything is supposed to be on a single line. 👀